.travis.yml

```diff
@@ -12,8 +13,7 @@ env:
     - ANDROID_BUILD_TOOLS_VERSION=28.0.3
 
 language: node_js
-node_js: 
-  - "6"
+node_js: 6
```

Comment:
I'm lazy, see my comment here: https://github.com/apache/cordova-browser/pull/76#pullrequestreview-232981486 :)

Reply:
Both works, here I decided to go what @erisu always uses because it looks more like "this is the version" instead of "this is the list of versions" - string vs. int I don't really care as it is all the same in YAML.